File: javascript/examples/get_documents_bulk_download.js
const baseRequest = require('../base_request.js').baseRequest
const path = require('path');
process.chdir(path.resolve(__dirname, '../../'));
require('dotenv').config();

// Example of GET /documents/bulk-download

const DOCUMENT_ID = process.env.DOCUMENT_ID;

params = {
  ids: [DOCUMENT_ID]
}

token = baseRequest.request.token({ path: '/documents/bulk-download' , queryParams: params});

baseRequest.request.get({ path: '/documents/bulk-download', token: token, queryParams: params});

File: python/examples/get_documents_bulk_download.py
import pprint
import os
import sys

sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

from request import BaseRequest
from dotenv import dotenv_values

config = dotenv_values()

# Example GET /documents/bulk-download

class GetDocumentsBulkDownloadRequest(BaseRequest):
    def __init__(self):
        super().__init__()

    def bulk_download_documents(self, company_ids):
        params = {
            'ids': document_ids
        }
        return self.make_request('GET', '/documents/bulk-download', params=params)

document_id  = config['DOCUMENT_ID']
document_ids = [document_id]

request = GetDocumentsBulkDownloadRequest()
response = request.bulk_download_documents(document_ids)

pprint.pprint(response)
